lib/Graph/Graph.cpp

```diff
@@ -1364,7 +1364,11 @@ BatchedReduceAddNode *
 Function::createBatchedReduceAdd(llvm::StringRef name, NodeValue batch,
                                  llvm::ArrayRef<unsigned_t> axes) {
   auto outDims = getNewShapeWithoutAxes(batch.dims(), axes);
-  auto OT = getParent()->uniqueType(batch.getType()->getElementType(), outDims);
+  auto bTy = batch.getType();
+  auto OT = isQuantizedElemKind(bTy->getElementType())
```

Comment:
could we use: Type::newShape(const Type &T, llvm::ArrayRef<size_t> dims), that takes care about quantization.